server/http/s2s_realtime.cpp

```
@@ -0,0 +1,947 @@
+        const std::string prompt = build_prompt(requested_instructions, requested_tools);
+        if (prompt.size() > kMaxInstructionsLength) {
+            emit_error("Instructions too long (max 32000 characters)", "instructions_too_long");
+            return;
+        }
+        if (requested_input_rate != input_rate_) {
+            input_rate_ = requested_input_rate;
+            input_resampler_ = audio::AudioResampler(input_rate_, voicechat_->input_sample_rate());
+        }
+        tools_ = std::move(requested_tools);
+        instructions_ = std::move(requested_instructions);
+        if (!prompt_applied_) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)